solutions/src/main/java/ru/sberbank/school/task02/Calculator.java

```diff
@@ -0,0 +1,50 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class Calculator implements FxConversionService {
+    private ExternalQuotesService provider;
+
+    Calculator(ExternalQuotesService externalQuotesService) {
+        this.provider = externalQuotesService;
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        List<Quote> quotes = provider.getQuotes(symbol);
+        quotes.sort((right, left)-> {
```

Comment:
WhitespaceAround: &#39;-&gt;&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1537987115>*